### ISSUE TITLE
Font/Icon rendering on scaled outputs

### ISSUE THREAD

**Issue by kleinreact** [swaywm/sway#1061]
There seem to be some rendering issues on multi-monitor setups, where one of the monitors is scaled. My setup looks as follows:
```
output DP-3 resolution 1920x1200 position 0,0 
output DP-4 resolution 1920x1200 position 1920,0
output eDP-1 resolution 2560x1440 position 3840,600 scale 2
```
However, on the non-scaled monitors the font and the icons look always look blurred. 

To produce the following example I used the following test setup: 

First I edited the config to set all monitors to non-scaled:
```
output DP-3 resolution 1920x1200 position 0,0 
output DP-4 resolution 1920x1200 position 1920,0
output eDP-1 resolution 2560x1440 position 3840,600
```
and opend pavucontrol on DP-4. Then I reversed the changes, reloaded the config and opend a second pavucontrol. The result cleary shows the difference. The window on the left (first opened, non-scaled) looks the right way, while the one on the right (second opened, scaled) is blurry. Also note that the icons are from a different source.

![example](https://cloud.githubusercontent.com/assets/15340601/22370038/927fb9d2-e48f-11e6-9d67-0cb160805de8.png)

Interestingly, if I now move both windows to the scaled monitor (eDP-1), the effect reverses as demonstrated in the following picture.

![example2](https://cloud.githubusercontent.com/assets/15340601/22370156/469081ae-e490-11e6-96ed-f5d63e3587c9.png)

I already tried all kinds of combinations, but on one of the montors the windows the output is always blurry.

The windows are gtk3 windows with native wayland enabled
```
sway version 0.11-71-gb3c0aa3 (2017-01-19, branch "HEAD")
wlc (commit 12ee978)
gtk+ (v3.22.5)
```

**Comment by Drakulix** [comment]
I would assume one window gets rendered with the non-scaled resolution looking blurred on the scaled monitor and the scaled window gets downscaled for rendering on the non-scaled monitor making it looked blurred there.
So basically the monitor you open the window one determines the scale of the rendering.

You can force certain scaling for gtk apps with the GDK_SCALE environment variable. By default or when you set it to 0, it tries to automatically determine the scale. You can force a certain scale by setting it to 1 or 2.
This might help to confirm this theory.

My guess would be that this is a bug in `wlc`, which would need to change the scale of an open window somehow. I am not actually sure if/how this is possible.

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unreleased sway 1.0.
